tests/tck/features/match/SeekById.feature

```diff
@@ -220,51 +243,35 @@ Feature: Match seek by id
       """
       MATCH (v)
       WHERE NOT id(v) == 'Paul Gasol'
-      RETURN v.name AS Name, v.age AS Age
+      RETURN v.player.name AS Name, v.player.age AS Age
       """
     Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
     When executing query:
       """
       MATCH (v)
       WHERE NOT id(v) IN ['James Harden', 'Jonathon Simmons', 'Klay Thompson', 'Dejounte Murray']
-      RETURN v.name AS Name
-      """
-    Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
-    When executing query:
```

Comment:
Why delete this?

Reply:
can select the index by introducing tag, this case can get the result, already move this case to line :153

Reply:
Could change to :
```
      MATCH (v)
      WHERE id(v) IN ['James Harden', 'Jonathon Simmons', 'Klay Thompson', 'Dejounte Murray']
      OR v.age == 23
      RETURN v.name AS Name
```
This is aim to test or logical.

Reply:
ok